packages/desktop-gui/cypress/fixtures/release_notes.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title": "Check Out This Release",
+  "version": "1.2.3",
+  "bannerImage": "https://placekitten.com/1000/200",
+  "content": "<h1>This is a great release</h1><h2>dolor sit amet</h2><h3>consectetur adipiscing elit</h3><h4>sed do eiusmod </h4><h5>tempor incididunt</h5><h6>ut labore et</h6><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod \ntempor incididunt ut labore et dolore magna aliqua. </p><ul><li><p><b>bold</b> text</p></li><li><p><i>italic</i> text</p><ul><li><p><a href=\"https://placekitten.com/1000/200\">link</a><u></u></p></li><li><p><u>underlined</u> text</p></li></ul></li></ul><ul><li><p>some <code>code</code> here</p></li></ul><ol><li><p>one</p></li><li><p>two</p><ol><li><p>aaay</p></li><li><p>beee</p></li></ol></li><li><p>three</p></li></ol><blockquote><p>Here&#39;s a blockquote</p></blockquote><hr/><p>Duis aute irure dolor in <a href=\"https://placekitten.com/1000/200\">reprehenderit</a> in voluptate \nvelit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint \noccaecat cupidatat non proident, sunt in culpa qui officia deserunt \nmollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod \ntempor incididunt ut labore et dolore magna aliqua. </p><p></p>",
```

Comment:
I remember emojis being a giant nightmare when integrating with contentful in the docs banner. May not be a concern here, but maybe we should look into testing this. LOOK AT THESE TESTS 😆  https://github.com/cypress-io/cypress-documentation/blob/develop/cypress/integration/contentful_banners_spec.js#L18:L18

Reply:
Good call. Added tests for emoji as well as testing it out manually and it all seems to work fine without any extra effort.